packages/metal-web-component/test/define_web_component.js

```diff
@@ -0,0 +1,234 @@
+'use strict';
+
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+import UA from 'metal-useragent';
+import { JSXComponent } from 'metal-jsx';
+
+import { defineWebComponent } from '../src/define_web_component';
+
+describe('Web components', function() {
+	let el;
+
+	before(function() {
+		if (UA.matchUserAgent('MSIE') || isSafariVersion('8.0')) {
```

Comment:
Correct, it would be nice to have something in `UA` that simply returns the major version number of the browser, regardless of what browser it is. We have a method in Portal that does something like that.